CMakeLists.txt

```diff
@@ -104,6 +104,10 @@ option(QUIC_DISABLE_POSIX_GSO "Disable GSO for systems that say they support it
 option(QUIC_OFFICIAL_RELEASE "Configured the build for an official release" OFF)
 set(QUIC_FOLDER_PREFIX "" CACHE STRING "Optional prefix for source group folders when using an IDE generator")
 set(QUIC_LIBRARY_NAME "msquic" CACHE STRING "Override the output library name")
+set(QUIC_ENABLE_LOGGING_STDOUT 0 CACHE STRING "Enables logging on stdout at given verbosity (0=off, 4=debug)")
```

Comment:
Why go this route, and not just on/off? And later, we could add a `LOGGING_LEVEL` param if we want?

Reply:
I revert to on/off if you prefer.